templates/zerver/help/analytics.md

```diff
@@ -0,0 +1,35 @@
+# Analytics
+
+Zulip provides analytics for tracking various usage statistics about you and your organization.
+So, you can get a holistic view of how your organization's communications.
```

Comment:
Maybe, "holistic view of your organization's communications" because leaving "how" in is grammatically incorrect.